src/assets/styles/settings/variables.scss

```diff
@@ -20,17 +18,12 @@ $color-error-bg: #ffffff;
 $color-success: #000000;
 $color-success-bg: #000000;
```

Comment:
How many of these other variables are used? Can we clean them up? Maybe thats a seperate PR...

Reply:
Yeah I noticed this as well. I planned on doing this in a separate PR - will follow-up!

Reply:
@tauthomas01 had adjusted some of these in a past PR. These default colors need to be redone... success state is black text on a black background???